frontend/components/Logo.js

```diff
@@ -0,0 +1,49 @@
+// import React from "react";
```

Comment:
Same here, I think this file is no longer being used (double check!), so you can get rid of it so there's no confusion for future devs.

Reply:
And I think once you get rid of this file, you can also get rid of the logo.scss file below.